js/app.js

```diff
@@ -4,6 +4,68 @@ $(document).ready( function() {
   // check to make sure JS is loaded
   console.log('JS is loaded!');
 
-  // your code here
 
+  $('.input-form').on('submit', function(event) {
```

Comment:
Hey Connor,

This code looks awesome - I don't even have anything to nit-pick over. Keep up the good work!

Matt